### ISSUE TITLE
Masking of adjacency matrix

### ISSUE THREAD

**Issue by kochkinaelena** [Diego999/pyGAT#34]
Hi,

It looks like the code is processing a single adjacency matrix that includes information about connections between graph nodes in training, development and testing sets. I was wondering where in the code the adjacency matrix is masked such that it is not using the parts that reflect the training set during testing?

Many thanks,
Elena

**Comment by jiangxinke** [comment]
I wonder too. I think the author forget to perform "mask" on GAT processing. Then the features for test set nodes will aggregate on train node.

**Comment by 2578562306** [comment]
This code is here"       attention = torch.where(adj > 0, e, zero_vec)" 
通过这个where方法实现的大于0使用e小雨则是zero。成功实现了mask attention

**Comment by 2578562306** [comment]
> I wonder too. I think the author forget to perform "mask" on GAT processing. Then the features for test set nodes will aggregate on train node.

This code is here" attention = torch.where(adj > 0, e, zero_vec)"
通过这个where方法实现的大于0使用e小雨则是zero。成功实现了mask attention

**Comment by jiangxinke** [comment]
@2578562306  Yes, you are right.
